.gitignore

```diff
@@ -24,3 +24,6 @@ _local_/*
 
 # Ignore python editable installs
 *.egg-info/
+
+# Python venvs
+.venv/
```

Comment:
why would you set up your venv hidden in the root directory?

Reply:
I thought it may be useful to allow for a venv for dawn (that's how I develop for dawn). If you don't agree or think it should be in a personal global .gitignore file, then I can remove from here.

Reply:
I mean I also use a venv for dawn. But i feel like the hidden folder in the source directory is an odd choice and would therefore not put it here

Reply:
I agree now that this should be in a users global .gitignore, so I removed it!